spec/features/movies/show_spec.rb

```diff
@@ -0,0 +1,87 @@
+require "rails_helper"
+
+RSpec.describe "/movies/:id" do
+  describe "as a visitor" do
+    describe "when I visit a movies show page" do
+      # As a user,
+      # When I visit a movie's show page.
+      # I see the movie's title, creation year, and genre,
+      # and a list of all its actors from youngest to oldest.
+      # And I see the average age of all of the movie's actors
+      it "displays the movies title, creation year, and genre" do
+        studio_1 = Studio.create!(name: "Universal Studios", location: "Hollywood")
+        studio_2 = Studio.create!(name: "Janbarra Studios", location: "Somewhere Else")
+
+        movie_1 = studio_1.movies.create!(title: "Wayne's World", creation_year: "1992", genre: "Amazing")
+        movie_2 = studio_1.movies.create!(title: "Bill and Ted's Excellent Adventure", creation_year: "2000", genre: "Action")
+        movie_3 = studio_2.movies.create!(title: "Passion of the Christ", creation_year: "1", genre: "Horror")
+
+        visit "/movies/#{movie_1.id}"
+
+        expect(page).to have_content("Movie Title: #{movie_1.title}")
+        expect(page).to have_content("Year Created: #{movie_1.creation_year}")
+        expect(page).to have_content("Genre: #{movie_1.genre}")
```

Comment:
These are nice robust assertions that associate the movie attribute with the title